src/extensions/default/bramble-move-file/MoveToDialog.js

```diff
@@ -114,9 +114,7 @@ define(function (require, exports, module) {
             MoveUtils.move(source, destination)
             .done(_finishMove.bind(null, source, destination))
             .fail(_failMove.bind(null, source, destination))
-            .always(function() {
-                dialog.close();
-            });
+            .then(dialog.close());
```

Comment:
I tried 2 of your suggestions and none would work. Also, `done` will work just like `then`.